application-task-ui/src/main/resources/TaskManager/KanbanBoardMacro.xml

```diff
@@ -43,6 +43,8 @@
 * ##user## [optional, default=$xcontext.userReference] define the user for who tasks are assigned
 * ##space## [optional, default=TaskManager] define the space to look for tasks
 * ##project## [optional, default=] define the project to look for tasks
+* ##colors## [optional, default=green,blue,orange] define the colors of the Kanban columns
+* ##columns## [optional, default=ToDo,InProgress,Done] define the statuses to display as Kanban columns
```

Comment:
Where are the default values defined?

Reply:
Defaults are defined [here](https://github.com/xwikisas/application-task/blob/f3af00e93071d5753119662429d6cb61a63f6c84/application-task-ui/src/main/resources/TaskManager/KanbanBoardMacro.xml#L907) for columns (and [here](https://github.com/xwikisas/application-task/blob/f3af00e93071d5753119662429d6cb61a63f6c84/application-task-ui/src/main/resources/TaskManager/KanbanBoardMacro.xml#L985) for the colors). The parameters were already defined, so I only added them to the documentation.